dodgr.Rmd

```diff
@@ -93,6 +95,19 @@ Plot
 plot(paths2)
 ```
 
+# On contracted graph
+
+```{r}
+o <- match_pts_to_verts(dodgr_vertices(graph), origins, connected = TRUE)
+d <- match_pts_to_verts(dodgr_vertices(graph), destinations, connected = TRUE)
```

Comment:
```suggestion
d <- v$id[match_pts_to_verts(v, destinations, connected = TRUE)]
```
Sorry, that was my bad - those `o` and `d` values should be vertex IDs. I've also updated code to issue message when trying to do the former way.

Reply:
Ace, thanks! Will try now. Error could be a bit more informative, right?